scripts/generate-config.sh

```diff
@@ -39,12 +39,22 @@ compare_extensions=(
   yaml
   go
   proto_descriptor
+  html
 )
 
+exit_with=0
+echo "Generating config against istio $(cd ${istio_src} && git describe --contains --all HEAD)"
 for i in "${compare_extensions[@]}" ; do
   for file in "${target_dir}/config/"*.${i} ; do
-    diff -b "$file" "config/${file##*/}"
+    changed=$(diff -b "$file" "${root_dir}/config/${file##*/}")
```

Comment:
This went under my radar initially. I think we might want to notice as well when there are space changes - those could be semantic changes depending on the context, and since we don't have a parser, I'd say it's best for us to error out on space changes for review.